src/parsing/standard/tax.ts

```diff
@@ -1,15 +1,17 @@
 import { StringDict } from "../common";
 import { Field } from "./field";
+import { BaseFieldConstructor } from "./base";
 import { floatToString } from "./amount";
 
-interface TaxConstructor {
-  prediction: StringDict;
-  valueKey?: string;
+/**
+ * @property {string} constructor.rateKey - Key to use to get the tax rate in the prediction dict.
+ * @property {string} constructor.codeKey - Key to use to get the tax code in the prediction dict.
+ * @property {string} constructor.baseKey - Key to use to get the base tax in the prediction dict.
+ */
+interface TaxConstructor extends BaseFieldConstructor {
   rateKey?: string;
   codeKey?: string;
   baseKey?: string;
-  reconstructed?: boolean;
-  pageId?: number;
```

Comment:
Can't we have document-level taxes?

Reply:
it inherits from the base params which have these properties

Reply:
Oh my bad I misread the diff cuz it cut off right below and started again with `pageId!: number;` so I thought you had enforced it, but it's only for the `TaxField` which makes more sense.